notebooks/import.py

```diff
@@ -240,11 +240,13 @@ def create_points(cursor, collection_id, histogram):
 
 
 def create_stat(cursor, dataset_id, metric_id, population, subgroup, key,
-                value):
+                value, low=None, high=None, level=None):
```

Comment:
Probably makes sense to specify `confidence_*` -- it's not clear from the context of just the method signature what low/high/level might mean

Reply:
:+1: